oauthproxy.go

```diff
@@ -645,6 +638,13 @@ func (p *OAuthProxy) SignInPage(rw http.ResponseWriter, req *http.Request, code
 		redirectURL = "/"
 	}
 
+	if err := p.ClearSessionCookie(rw, req); err != nil {
+		logger.Printf("Error clearing session cookie: %v", err)
+		p.pageWriter.WriteSignInPage(rw, req, redirectURL, code)
+		return
```

Comment:
is this still necssary? do you need to return here. cannot you just remove those lines?
```suggestion
```

Reply:
Oh, I completely overlooked that.
I'll check it out. ;)